pkg/handler/pipeline.go

```diff
@@ -1994,3 +1994,20 @@ func (h *PublicHandler) ListComponentRuns(ctx context.Context, req *pb.ListCompo
 
 	return resp, nil
 }
+
+// todo: rename function to ListPipelineRunsByRequester in protobuf and update message names here
+func (h *PublicHandler) ListPipelineRunsByCreditOwner(ctx context.Context, req *pb.ListPipelineRunsByCreditOwnerRequest) (*pb.ListPipelineRunsByCreditOwnerResponse, error) {
```

Comment:
👀 I guess the method rename to `ByRequester` (or, ideally, `Search`) will be done in a separate PR @joremysh ?